year-pages/2019-conferences-list.md

```diff
@@ -2,6 +2,8 @@
 
 | Conference | Date | Venue | Description | Scholarship |
 |------------|------|-------|-------------|-------------|
+| [ReactFoo](https://reactfoo.in/2019/) | 1-2 Mar | NIMHANS Convention Centre, Bangalore | India’s largest React conference for JavaScript and front-end engineers, cross-platform developers. | No |
 | [SACON 2019](https://www.sacon.io/) | 15-16 Feb | Bengaluru |  India’s 1st Security Architecture Conference | No
 | [RubyConf India](https://www.rubyconfindia.org/) | 20-21 Jan | Resort Rio and Club Rio Royale, Goa | A global event complementing other RubyConf events across the world. This year it's a single track 2 day event, focused on the Language, Framework and Tools. | [Yes](https://www.rubyconfindia.org/scholarship/) |
+| [GOPHERCON](https://gopherconindia.com/) | 18-19 Jan | Resort Rio, Goa | Go is an open source project developed by a team at Google and many contributors from the open source community. | No |
```

Comment:
Can you update the following:

- title: `GopherCon India`

- description: `Go conference in India. Go is an open source project developed by a team at Google and many contributors for the open source community.`

- Scholarship: Yes, and link to https://gopherconindia.com/scholarship/.